tests/integration/update_cluster/aws-lb-controller/data/aws_launch_template_master-us-test-1a.masters.minimal.example.com_user_data

```diff
@@ -121,6 +121,8 @@ ensure-install-dir
 
 cat > conf/cluster_spec.yaml << '__EOF_CLUSTER_SPEC'
 cloudConfig:
+  awsEBSCSIDriver:
+    enabled: false
```

Comment:
I'm wondering if we can exclude these fields altogether if the driver isn't enabled, and have the consumer of this data check for non nil. That way we aren't encroaching on userdata space